java/metrics/src/main/scala/org/typelevel/otel4s/java/metrics/ObservableCounterBuilderImpl.scala

```diff
@@ -55,5 +55,28 @@ private[java] case class ObservableCounterBuilderImpl[F[_]](
         })
         .as(new ObservableCounter {})
     )
-
+  def create(
```

Comment:
> I was looking for some DRY-ness, but there is probably a better way. The right thing would probably be to refactor the java builders.

As an alternative, we can introduce a private `create` method to reuse some code:
```scala
private def createInternal(cb: ObservableLongMeasurement => F[Unit]) =
  Dispatcher.sequential.flatMap(dispatcher =>
      ...
  )
  .as(new ObservableCounter {})
```

Then, public methods look cleaner:
```scala
def createWithCallback(
    cb: ObservableMeasurement[F, Long] => F[Unit]
): Resource[F, ObservableCounter] =
  createInternal(counter => cb(new ObservableLongImpl(counter)))
  
def create(
    measurements: F[List[Measurement]]
): Resource[F, ObservableCounter] =
  createInternal(counter =>
    measurements.flatMap(ms =>
      F.delay(
        ms.foreach(m =>
          counter.record(m.value, Conversions.toJAttributes(m.attributes))
        )
      )
    )
  )
```

Reply:
Yea, but I don't think it will work across instruments with different types.

Reply:
Ok. It's a tiny bit cleaner. Thanks for the patience :)